### PR TITLE
(maint) Use yum to install build dependencies on AIX

### PR DESCRIPTION
**Pull request by geoffnichols**
On AIX, bootstrap and use yum to install build dependencies.

Replaces https://github.com/puppetlabs/puppet-runtime/pull/75

### CODE REVIEW

**Comment by geoffnichols** [comment]
Rebased against https://github.com/puppetlabs/puppet-runtime/pull/77.

File: configs/platforms/aix-6.1-ppc.rb [hunk 1]
```diff
@@ -34,12 +22,24 @@
...
+  plat.provision_with "echo '[AIX_Toolbox_noarch_mirror]\nname=AIX Toolbox noarch repository\nbaseurl=https://artifactory.delivery.puppetlabs.net/artifactory/rpm__remote_aix_linux_toolbox/RPMS/noarch/\ngpgcheck=0' > /opt/freeware/etc/yum/repos.d/toolbox-noarch-mirror.repo"
+  plat.provision_with "echo '[AIX_Toolbox_61_mirror]\nname=AIX 61 specific repository\nbaseurl=https://artifactory.delivery.puppetlabs.net/artifactory/rpm__remote_aix_linux_toolbox/RPMS/ppc-6.1/\ngpgcheck=0' > /opt/freeware/etc/yum/repos.d/toolbox-61-mirror.repo"
+
+  # Install build dependencies
+  plat.provision_with "yum install -y coreutils autoconf gawk-3.1.3-1 pkg-config-0.19-6 rsync sed make tar glib2 perl zlib-1.2.3-4 zlib-devel-1.2.3-4"
+
+  # For now, use pl-cmake from pl-build-tools. Work around harcoded path in pl-cmake toolchain file.
+  #plat.provision_with "/opt/freeware/bin/sed -i 's|/opt/pl-build-tools|/opt/freeware|g' /opt/pl-build-tools/pl-build-toolchain.cmake"
```
Comment (caseywilliams) [inline configs/platforms/aix-6.1-ppc.rb]: This all looks great, but let's get rid of this comment

Comment (geoffnichols) [reply configs/platforms/aix-6.1-ppc.rb]: Yep, good catch!

Comment (geoffnichols) [reply configs/platforms/aix-6.1-ppc.rb]: Updated to remove the comment

**Comment by geoffnichols** [comment]
Updated to use yum to install pl-build-tools and squash commits

**Pull request closed without merging**